vlm/src/main/scala/geotrellis/contrib/vlm/geotiff/GeoTiffResampleRasterSource.scala

```diff
@@ -50,29 +50,31 @@ case class GeoTiffResampleRasterSource(
     }
   }
 
+  private[geotiff] val options = parentOptions.copy(rasterExtent = Some(rasterExtent))
+
   @transient protected lazy val closestTiffOverview: GeoTiff[MultibandTile] =
     tiff.getClosestOverview(rasterExtent.cellSize, strategy)
 
-  def reproject(targetCRS: CRS, reprojectOptions: Reproject.Options, strategy: OverviewStrategy): GeoTiffReprojectRasterSource =
-    new GeoTiffReprojectRasterSource(uri, targetCRS, reprojectOptions, strategy) {
+  override def reproject(targetCRS: CRS, reprojectOptions: Reproject.Options, strategy: OverviewStrategy): GeoTiffReprojectRasterSource =
```

Comment:
why have you added these verbose identifiers? O:

Reply:
Oh, I think I added those when I was first trying things out. I just forgot to clean them up. Thanks for catching that :+1: